src/c/c_client.cpp

```diff
@@ -679,6 +795,41 @@ SRError get_model(void* c_client,
   return result;
 }
 
+// Put a script in the database that is stored in a file in a multi-GPU system
```

Comment:
Please be consistent in ordering all the multiGPU functions after their regular  (non-GPU) versions.  Currently, there's a mix with some before and some after

Reply:
Thanks for catching this. Double-checked both C/Fortran and it seems to be just `set_script_from_file_multigpu`